packages/xstate-inspect/src/types.ts

```diff
@@ -6,10 +6,13 @@ export type MaybeLazy<T> = T | (() => T);
 
 export type ServiceListener = (service: Interpreter<any>) => void;
 
+export type Replacer = (key: string, value: any) => any;
+
 export interface InspectorOptions {
   url: string;
   iframe: MaybeLazy<HTMLIFrameElement | null | false>;
   devTools: MaybeLazy<XStateDevInterface>;
+  serialize: Replacer | undefined;
```

Comment:
Usually, such types can be consumed by people - and making all of those fields required make it harder so. I would probably recommend making the optional things optional here and creating a separate that for "resolved" options that would only be used within our implementation